src/DIRAC/TransformationSystem/Client/test/Test_Client_WorkflowTasks.py

```diff
@@ -91,9 +91,11 @@
         (taskDictNoInputsNoSite, True, True, expectedBulk),
     ],
 )
-def test_prepareTranformationTasks(taskDictionary, bulkSubmissionFlag, result, expectedRes):
+def test_prepareTranformationTasks(mocker, taskDictionary, bulkSubmissionFlag, result, expectedRes):
+    mocker.patch("DIRAC.TransformationSystem.Client.WorkflowTasks.getDNForUsername", return_value=S_OK(["pippo"]))
```

Comment:
```suggestion
    mocker.patch("DIRAC.TransformationSystem.Client.WorkflowTasks.getDNForUsername", return_value=S_OK(["/bih/boh/DN"]))
```
Or something that cannot be confused with the username